TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Model/Workspace.php

```diff
@@ -560,7 +565,7 @@ protected function findNodeDataInTargetWorkspace(NodeInterface $node, Workspace
      * @return void
      * @Flow\Signal
      */
-    protected function emitBaseWorkspaceChanged(Workspace $workspace, Workspace $oldBaseWorkspace, Workspace $newBaseWorkspace)
+    protected function emitBaseWorkspaceChanged(Workspace $workspace, Workspace $oldBaseWorkspace = null, Workspace $newBaseWorkspace = null)
```

Comment:
Ah right, thanks for the explanation.
Regarding BC: I don't even think it's breaking because in your slot the exact method signature is not checked IIRC, it was merely a curious question